backend/src/packages/chats/chat.repository.ts

```diff
@@ -17,6 +20,25 @@ class ChatRepository implements Repository {
     return Promise.resolve(null);
   }
 
+  public async findById(id: number, userId: number): Promise<ChatEntity> {
+    const chat = await this.chatModel
+      .query()
+      .findById(id)
+      .withGraphJoined(ChatsRelation.MEMBERS)
+      .whereExists(UserToChatModel.query().where('userId', userId))
+      .orderBy('createdAt', 'DESC')
```

Comment:
It will be better to create a sort enum for this

Reply:
We already have it, `SortType` as I remember. Just use it